### ISSUE TITLE
Adding a YAML based policy via API

### ISSUE THREAD

**Issue by veeral-patel** [cerbos/cerbos#1725]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

I'm wondering how I can add a YAML based policy via the create policy admin API.

https://docs.cerbos.dev/cerbos/latest/api/admin_api#_addupdate_policies

The request in the docs above shows a JSON request body. If I have a YAML policy, do I need to convert into JSON first? Is this what Cerbos Playground does?

**Comment by charithe** [comment]
Yes, the policy needs to be in JSON format. The `cerbosctl put` utility and some of our SDKs handle the conversion for you automatically.

**Comment by veeral-patel** [comment]
Thanks!
